<!DOCTYPE html>
<html lang='en'>
<head>
<title>musl-hacks.patch\chromium\community - aports - Unnamed repository; edit this file 'description' to name the repository.
</title>
<meta name='generator' content='cgit v1.2.1'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit.css'/>
<link rel='shortcut icon' href='//alpinelinux.org/alpine-logo.ico'/>
<link rel='alternate' title='Atom feed' href='http://git.alpinelinux.org/aports/atom/community/chromium/musl-hacks.patch?h=master' type='application/atom+xml'/>
<link rel='vcs-git' href='git://git.alpinelinux.org/aports' title='aports Git repository'/>
<link rel='vcs-git' href='https://git.alpinelinux.org/aports' title='aports Git repository'/>
</head>
<body>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/'><img src='//wiki.alpinelinux.org/images/alogo.png' alt='cgit logo'/></a></td>
<td class='main'><a href='/'>index</a> : <a title='aports' href='/aports/'>aports</a></td><td class='form'><form method='get'>
<select name='h' onchange='this.form.submit();'>
<option value='1.10-stable'>1.10-stable</option>
<option value='1.9'>1.9</option>
<option value='2.0-stable'>2.0-stable</option>
<option value='2.1-stable'>2.1-stable</option>
<option value='2.2-stable'>2.2-stable</option>
<option value='2.3-stable'>2.3-stable</option>
<option value='2.4-stable'>2.4-stable</option>
<option value='2.5-stable'>2.5-stable</option>
<option value='2.6-stable'>2.6-stable</option>
<option value='2.7-stable'>2.7-stable</option>
<option value='3.0-stable'>3.0-stable</option>
<option value='3.1-stable'>3.1-stable</option>
<option value='3.10-stable'>3.10-stable</option>
<option value='3.11-stable'>3.11-stable</option>
<option value='3.12-stable'>3.12-stable</option>
<option value='3.2-stable'>3.2-stable</option>
<option value='3.3-stable'>3.3-stable</option>
<option value='3.4-stable'>3.4-stable</option>
<option value='3.5-stable'>3.5-stable</option>
<option value='3.6-stable'>3.6-stable</option>
<option value='3.7-stable'>3.7-stable</option>
<option value='3.8-stable'>3.8-stable</option>
<option value='3.9-stable'>3.9-stable</option>
<option value='master' selected='selected'>master</option>
</select> <input type='submit' value='switch'/></form></td></tr>
<tr><td class='sub'>Unnamed repository; edit this file 'description' to name the repository.
</td><td class='sub right'>uwsgi</td></tr></table>
<table class='tabs'><tr><td>
<a href='/aports/about/'>about</a><a href='/aports/'>summary</a><a href='/aports/refs/'>refs</a><a href='/aports/log/community/chromium/musl-hacks.patch'>log</a><a class='active' href='/aports/tree/community/chromium/musl-hacks.patch'>tree</a><a href='/aports/commit/community/chromium/musl-hacks.patch'>commit</a><a href='/aports/diff/community/chromium/musl-hacks.patch'>diff</a><a href='/aports/stats/community/chromium/musl-hacks.patch'>stats</a></td><td class='form'><form class='right' method='get' action='/aports/log/community/chromium/musl-hacks.patch'>
<select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='search' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/aports/tree/'>root</a>/<a href='/aports/tree/community'>community</a>/<a href='/aports/tree/community/chromium'>chromium</a>/<a href='/aports/tree/community/chromium/musl-hacks.patch'>musl-hacks.patch</a></div><div class='content'>blob: 45d938b78fda96b6dc2dfcb7b590c48dffbc44f6 (<a href='/aports/plain/community/chromium/musl-hacks.patch'>plain</a>) (<a href='/aports/blame/community/chromium/musl-hacks.patch'>blame</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a id='n1' href='#n1'>1</a>
<a id='n2' href='#n2'>2</a>
<a id='n3' href='#n3'>3</a>
<a id='n4' href='#n4'>4</a>
<a id='n5' href='#n5'>5</a>
<a id='n6' href='#n6'>6</a>
<a id='n7' href='#n7'>7</a>
<a id='n8' href='#n8'>8</a>
<a id='n9' href='#n9'>9</a>
<a id='n10' href='#n10'>10</a>
<a id='n11' href='#n11'>11</a>
<a id='n12' href='#n12'>12</a>
<a id='n13' href='#n13'>13</a>
<a id='n14' href='#n14'>14</a>
<a id='n15' href='#n15'>15</a>
<a id='n16' href='#n16'>16</a>
<a id='n17' href='#n17'>17</a>
<a id='n18' href='#n18'>18</a>
<a id='n19' href='#n19'>19</a>
<a id='n20' href='#n20'>20</a>
<a id='n21' href='#n21'>21</a>
<a id='n22' href='#n22'>22</a>
<a id='n23' href='#n23'>23</a>
<a id='n24' href='#n24'>24</a>
<a id='n25' href='#n25'>25</a>
<a id='n26' href='#n26'>26</a>
<a id='n27' href='#n27'>27</a>
<a id='n28' href='#n28'>28</a>
<a id='n29' href='#n29'>29</a>
<a id='n30' href='#n30'>30</a>
<a id='n31' href='#n31'>31</a>
<a id='n32' href='#n32'>32</a>
<a id='n33' href='#n33'>33</a>
<a id='n34' href='#n34'>34</a>
</pre></td>
<td class='lines'><pre><code><span style="color:#000000; font-weight:bold">--- ./base/debug/stack_trace.cc</span>
<span style="color:#0057ae">+++ ./base/debug/stack_trace.cc</span>
<span style="color:#010181">&#64;&#64; -233,7 +233,7 &#64;&#64;</span>
 }
 std::string StackTrace::ToStringWithPrefix(const char* prefix_string) const {
   std::stringstream stream;
<span style="color:#000000; font-weight:bold">-#if !defined(__UCLIBC__) &amp;&amp; !defined(_AIX)</span>
<span style="color:#0057ae">+#if defined(__GLIBC__) &amp;&amp; !defined(_AIX)</span>
   OutputToStreamWithPrefix(&amp;stream, prefix_string);
 #endif
   return stream.str();
<span style="color:#000000; font-weight:bold">--- ./net/socket/udp_socket_posix.cc</span>
<span style="color:#0057ae">+++ ./net/socket/udp_socket_posix.cc</span>
<span style="color:#010181">&#64;&#64; -1191,7 +1191,7 &#64;&#64;</span>
     msg_iov-&gt;push_back({const_cast&lt;char*&gt;(buffer-&gt;data()), buffer-&gt;length()});
   msgvec-&gt;reserve(buffers.size());
   for (size_t j = 0; j &lt; buffers.size(); j++)
<span style="color:#000000; font-weight:bold">-    msgvec-&gt;push_back({{nullptr, 0, &amp;msg_iov[j], 1, nullptr, 0, 0}, 0});</span>
<span style="color:#0057ae">+    msgvec-&gt;push_back({{nullptr, 0, &amp;msg_iov[j], 1, 0, 0, 0}, 0});</span>
   int result = HANDLE_EINTR(Sendmmsg(fd, &amp;msgvec[0], buffers.size(), 0));
   SendResult send_result(0, 0, std::move(buffers));
   if (result &lt; 0) {
diff --git a/chrome/common/string_matching/fuzzy_tokenized_string_match.cc b/chrome/common/string_matching/fuzzy_tokenized_string_match.cc
index 8351fa7..7ace813 100644
<span style="color:#000000; font-weight:bold">--- ./chrome/common/string_matching/fuzzy_tokenized_string_match.cc</span>
<span style="color:#0057ae">+++ ./chrome/common/string_matching/fuzzy_tokenized_string_match.cc</span>
<span style="color:#010181">&#64;&#64; -6,6 +6,7 &#64;&#64;</span>
 
 #include &lt;algorithm&gt;
 #include &lt;iterator&gt;
<span style="color:#0057ae">+#include &lt;cmath&gt;</span>
 
 #include &quot;base/i18n/case_conversion.h&quot;
 #include &quot;base/metrics/field_trial_params.h&quot;
</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated by <a href='https://git.zx2c4.com/cgit/about/'>cgit v1.2.1</a> (<a href='https://git-scm.com/'>git 2.18.0</a>) at 2020-06-06 19:56:13 +0000</div>
</div> <!-- id=cgit -->
</body>
</html>
